### PR TITLE
Add "replayio open" command to launch browser in non-recording mode

### PR DESCRIPTION
**Pull request by bvaughn**
This has been requested so that people can do things like authentication (without Replay recording) before starting recording.

### CODE REVIEW

File: packages/replayio/src/utils/browser/launchBrowser.ts [hunk 1]
```diff
@@ -28,9 +29,10 @@ export async function launchBrowser(
   ];
   const processOptions = {
     env: {
-      RECORD_ALL_CONTENT: "1",
+      RECORD_ALL_CONTENT: record ? "1" : undefined,
+      RECORD_REPLAY_DONT_RECORD: record ? undefined : "1",
```
Comment (Andarist) [inline packages/replayio/src/utils/browser/launchBrowser.ts:33]: TIL, what is this? test plugins dont set this at all (IIRC)

Comment (Andarist) [reply packages/replayio/src/utils/browser/launchBrowser.ts:33]: sideways: do u know what `RECORD_REPLAY_DRIVER` is for? test plugins use it but we don't use it here

**Review by Andarist — APPROVED** [review]

File: packages/replayio/src/utils/browser/launchBrowser.ts (hunk 1 above)
Comment (bvaughn) [reply packages/replayio/src/utils/browser/launchBrowser.ts:33]: TIL too. Jason mentioned it in Discord. All I know about it is that if it's set, it tells the runtime not to record anything.

Comment (bvaughn) [reply packages/replayio/src/utils/browser/launchBrowser.ts:33]: > sideways: do u know what `RECORD_REPLAY_DRIVER` is for? test plugins use it but we don't use it here

No. Maybe we should remove it too? I don't remember why it's here. :)

**Pull request merged**